cli/cmd/suppressions.go

```diff
@@ -0,0 +1,57 @@
+//
+// Author:: Ross Moles (<ross.moles@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	// top-level cloud-account command
+	suppressionsCommand = &cobra.Command{
+		Use:     "suppressions",
+		Hidden:  true,
+		Aliases: []string{"suppression", "sup", "sups"},
```

Comment:
do we really need aliases like `sup` and `sups`? if this is a hidden command I don't think we do need so many aliases.

Reply:
But I am ok leaving them 😂 this is a nit

Reply:
probably not, but a the same time, why not? 🤷